packages/backend/src/modules/user/user.controller.ts

```diff
@@ -34,50 +47,75 @@ export class UserController {
         req.params.userId,
         updateUserCommand
       )
-      const dto: UserDto = this._mapper.mapToDto(updatedUser)
+      const dto: UserDTO = this._mapper.mapToDto(updatedUser)
       res.status(200).json(dto)
     } catch (e) {
-      if (e instanceof Error) {
-        console.log(
-          `UserController: error updating user with id ${req.params.userId}: ${e.message}`
-        )
+      if (e instanceof CannotFindDocumentError) {
+        console.error(`UserController.updateOneById: ${e.message}`)
+        console.error(e)
+        res.status(404).send('User not found')
+        return
+      }
+      if (e instanceof CannotCreateDocumentError) {
```

Comment:
that a update feature so why there is `CannotCreateDocumentError`?

Reply:
Yeah, this should be `CannotUpdateDocumentError`